packages/theme/src/theme/_navs.scss

```diff
@@ -33,12 +33,24 @@ $tc-tabbar-active-border-size: 0.2rem !default;
 				margin: 0;
 			}
 
+			.btn-link {
+				color: tokens.$coral-color-accent-text-weak;
+			}
+
+			&:focus,
 			&:hover,
+			&:active {
+				.btn-link {
+					// TODO: replace with token
+					background-color: rgba(255, 255, 255, 0.12);
+				}
+			}
+
 			&.active,
 			&.open {
 				.btn-link {
-					color: tokens.$coral-color-accent-text-weak;
-					background-color: tokens.$coral-color-accent-background-strong;
+					background-color: tokens.$coral-color-accent-text-weak;
+					color: tokens.$coral-color-accent-background-strong;
```

Comment:
This is strange to have a text color for a background and a background color for text 😄

Reply:
Right :smile: It was changed quickly for a quick check. I can find correct tokens